src/main/java/domain/Crew.java

```diff
@@ -0,0 +1,76 @@
+package domain;
+
+import domain.attendance.Attendance;
+import domain.attendance.AttendanceDate;
+import domain.attendance.AttendanceWarning;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.List;
+
+public class Crew implements Comparable<Crew> {
+    private final String name;
+    private final Attendance attendance;
+
+    public Crew(String name) {
+        this.name = name;
+        this.attendance = new Attendance(AttendanceDate.DEFAULT_START_DATE, LocalDate.now());
+
+        this.attendance.fillAttendanceDate();
+    }
+
+    public void initializeAttendance(List<LocalDateTime> localDateTimes) {
+        attendance.fillAttendanceDate();
+        localDateTimes.forEach(this::editAttendanceDateTime);
+    }
+
+    private void editAttendanceDateTime(LocalDateTime localDateTime) {
+        AttendanceDate attendanceDate = attendance.findAttendanceDate(localDateTime.toLocalDate());
+        attendanceDate.editDateTime(localDateTime);
+    }
+
+    public String getName() {
+        return this.name;
+    }
+
+    public Attendance getAttendance() {
+        return attendance;
+    }
+
+    public int countAbsence() {
+        return attendance.countAbsence();
+    }
+
+    public int countTardy() {
+        return attendance.countTardy();
+    }
+
+    public int countAbsenceIncludingTardy() {
+        return attendance.countAbsenceIncludingTardy();
+    }
+
+    public AttendanceDate findAttendanceDate(LocalDate localDate) {
+        return attendance.findAttendanceDate(localDate);
+    }
+
+    public boolean isAttendanceWarning() {
+        return AttendanceWarning.determineAttendanceWarning(this.attendance.countAbsenceIncludingTardy())
+                != AttendanceWarning.NONE;
+    }
+
+    @Override
+    public int compareTo(Crew compareCrew) {
+        if (this.attendance.countAbsence() < compareCrew.attendance.countAbsence()) {
+            return 1;
+        }
+        if (this.attendance.countAbsence() > compareCrew.attendance.countAbsence()) {
+            return -1;
+        }
+        if (this.attendance.countTardy() < compareCrew.attendance.countTardy()) {
+            return 1;
+        }
+        if (this.attendance.countTardy() > compareCrew.attendance.countTardy()) {
+            return -1;
+        }
+        return this.name.compareTo(compareCrew.name);
+    }
```

Comment:
아래와 같이 Comparator 를 활용해봐도 좋을 것 같아요!

```suggestion
    public int compareTo(Crew compareCrew) {
            return Comparator.comparingInt((Crew c) -> c.attendance.countAbsence()).reversed()
            .thenComparingInt(~~)
            .thenComparing(~~)
            .compare(~~);
    }
```

Reply:
Comparable에 대해서만 학습하고 사용해왔는데 덕분에 코드가 더 간결해진 것 같습니다. 좋은 기능 알려주셔서 감사합니다!